modules/api/src/main/java/org/apache/ignite/lang/ErrorGroups.java

```diff
@@ -500,6 +500,9 @@ public static class Network {
 
         /** Could not resolve address. */
         public static final int ADDRESS_UNRESOLVED_ERR = NETWORK_ERR_GROUP.registerErrorCode((short) 6);
+
+        /** Duplicate consistent ID. */
+        public static final int DUPLICATE_CONSISTENT_ID_ERR = NETWORK_ERR_GROUP.registerErrorCode((short) 7);
```

Comment:
Do we really need a new public error code? I mean, can exception with this code reach the user?